support/sier-codec/src/json.rs

```diff
@@ -5,7 +5,13 @@ use std::sync::Arc;
 pub fn parse_serde_value<'a>(value: &SerdeValue, type_: &'a Type) -> Result<Value<'a>, Error<'a>> {
     match value {
         SerdeValue::Null => unimplemented!(),
-        SerdeValue::Bool(_bool) => unimplemented!(),
+        SerdeValue::Bool(bool) => {
```

Comment:
Since `bool` is a keyword in Rust, it's weird to use as a variable name. Just `b` would be clear enough from context. Alternatively `v` for value.